### ISSUE TITLE
Npm wont' install latest version

### ISSUE THREAD

**Issue by tquiroga** [shoutem/ui#345]
I can't manage to get to install the latest version. It works for 0.21.x but not for 0.22.x
```
> npm install @shoutem/ui --save
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git checkout fbce4b9e478f808634a98dac48901ec6ed4ee9fd
npm ERR! fatal: reference is not a tree: fbce4b9e478f808634a98dac48901ec6ed4ee9fd
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/thomas/.npm/_logs/2017-12-19T18_36_19_995Z-debug.log
```
What could cause that?

**Comment by Vladimir-Vdovic** [comment]
Hi @tquiroga,
I've tried to replicate this on both OSX and Windows, but without success.

`fbce4b9e478f808634a98dac48901ec6ed4ee9fd` is the commit we've referenced to Goshakkk's `react-native-navigation-experimental-compat` repo, seen [here](https://github.com/goshakkk/react-native-navigation-experimental-compat/pull/3/commits/fbce4b9e478f808634a98dac48901ec6ed4ee9fd).

This may be an issue with your environment, namely npm and/or git.
